cala-server/src/graphql/primitives.rs

```diff
@@ -53,6 +72,18 @@ impl From<String> for TAG {
     }
 }
 
+impl AsRef<str> for TAG {
+    fn as_ref(&self) -> &str {
```

Comment:
why is this exposed?

Reply:
instead of making pub(super) i think it makes much more sense to call ```as_ref()``` during the try_from conversion